docs/tac-skills.md

```diff
@@ -9,7 +9,6 @@ There are two types of agents:
 
```

Comment:
we should add the `aea add connection oef` since it is not the default anymore.
Also `aea run --connections oef` or alternatively `aea config set.default_connection oef`

Reply:
I am doing that below actually